docs/installation.rst

```diff
@@ -31,7 +31,7 @@ Or add this to your composer.json configuration:
 
 	{
 		"require" : {
-			"FriendsOfCake/crud": "~4.0"
+			"FriendsOfCake/crud": "~4.2"
```

Comment:
Because if we go to version 5, that will be the stable version. But your project actually depends on version 4